packages/forma-36-react-components/src/components/Asset/Asset.tsx

```diff
@@ -23,6 +24,36 @@ export function isAssetType(type: string): type is AssetType {
   return Object.keys(types).includes(type);
 }
 
+const renderImage = (src: string, title: string) => (
+  <React.Fragment>
```

Comment:
Is there a difference between `<>` and `<React.Fragment>`? Or is a matter of personal preference?

If it's just preference, should we have a code styling convention?

Reply:
There's no difference. Most people think `<>` look better, but `<Fragment />` can lead to less issues from switching JSX pragma.

Reply:
I had to google JSX pragma
But at a first glance, it does not look like something we need to worry about
In any case, it would be nice to have a lint rule or something to force us to use only one way in the codebase

I like only `<>` because it's less typing honestly 🤷

Reply:
It would mostly be relevant if we used something like Emotion. Agreed it would be nice if we could lint it.